src/main/scala/ba/sake/hepek/bulma/component/CardComponents.scala

```diff
@@ -0,0 +1,66 @@
+package ba.sake.hepek.bulma.component
+
+import ba.sake.hepek.bulma._
+import scalatags.Text.all._
+
+object CardComponents extends CardComponents
+
+trait CardHeader extends BulmaElement {
+
+  def contentBuilder(attributeClass: BulmaModifier*)(title: String, iconClass: Option[String]) =
```

Comment:
I have a preference to push all `Option`-al parameters to the right and give them default value of `None`.  
You don't have to change it, it's just nicer (to me) not having to pass the parameter... :)

Reply:
I tried to align my commits according to this convention :smiley: